internal/subshell/bash/bash.go

```diff
@@ -219,3 +220,17 @@ func (v *SubShell) IsAvailable() bool {
 	}
 	return fileutils.FileExists(rcFile)
 }
+
+func (v *SubShell) TurnOffEcho() {
+	if runtime.GOOS == "windows" {
+		return // not supported
+	}
+	termecho.Off()
+}
+
+func (v *SubShell) TurnOnEcho() {
+	if runtime.GOOS == "windows" {
+		return // not supported
+	}
+	termecho.Off()
```

Comment:
Shouldn't these, and the other `TurnOnEcho` functions be calling `termecho.On()`?

Reply:
Oh, how embarrassing. Thanks for catching this.